HW0/P6/P6B.py

```diff
@@ -6,6 +7,20 @@
 def burnTime(t):
     time.sleep(t)
 
```

Comment:
The below functions are great, but they don't really solve the problem of code duplication (the only difference between the 2 functions is the 1-2 lines actually calling the wait time either in serial or in parallel). I think there could be a way to write a `getGeneralTime()` function and pass in some parameter to indicate serial vs parallel, to make this code even cleaner and to allow it to work beyond these two examples if ever it was necessary.